_articles/touchpad.md

```diff
@@ -21,3 +21,13 @@ Press <kbd>Fn</kbd>+<kbd>F1</kbd> to turn your laptop touchpad on/off.
 To configure your touchpad, press the Super Key <kbd><span class="fl-ubuntu"></span></kbd>, <kbd><span class="fl-pop-key"></span></kbd> then type <u>Mouse & Touchpad</u> and click on the result. The touchpad settings are under the **Touchpad** tab.
 
 ![Mouse Search](/images/touchpad/mouse.png)
+
+<u>GNOME Tweaks</u> has a few other options for the touchpad like disabling it while typing, the acceleration profile and mouse click emulation.
+
+![GNOME Tweaks](/images/touchpad/gnome-tweak-tool.png)
+
+<u>GNOME Tweaks</u> can be installed from the <u>Pop!_Shop</u> (Pop!_OS), <u>Ubuntu Software Center</u> (Ubuntu) or from the <u>Terminal</u> with this command:
```

Comment:
This one might be an exception because the `_` in Pop really upsets markdown

```suggestion
_GNOME Tweaks_ can be installed from the _Pop!_Shop_ (Pop!_OS),_Ubuntu Software Center_(Ubuntu) or from the_Terminal_ with this command:
```


```suggestion
_GNOME Tweaks_ can be installed from the <u>Pop!_Shop</u> (Pop!_OS),_Ubuntu Software Center_(Ubuntu) or from the_Terminal_ with this command:
```